packages/framework/fluid-static/src/compatibilityConfiguration.ts

```diff
@@ -40,5 +43,7 @@ export const compatibilityModeRuntimeOptions: Record<
 		// The runtime ID compressor is a prerequisite to use SharedTree but is off by default and must be explicitly enabled.
 		// It introduces a new type of op which is not compatible with 1.x clients.
 		enableRuntimeIdCompressor: "on",
+		// Enable GC Sweep going forward
+		gcOptions: { enableGCSweep: true },
```

Comment:
I don't think we should do this for 2.0. This means that anyone who uses 2.0 will have sweep enabled. We don't want that right? We may do that in 3.0.

Reply:
Why not?

Reply:
Discussed offline.  No rush here, and when we switch Sweep to default on we should do it across the whole codebase (update GC itself to default to on, in the absence of any runtime options)